src/common/transformers/UNIQ.spec.js

```diff
@@ -0,0 +1,25 @@
+import expect from 'expect';
+
+import { uniqArray } from './UNIQ';
+
+describe('UNIQ', () => {
+    it('should return uniq value', () => {
+        expect(uniqArray(['hello', 'hello'])).toEqual(['hello']);
+    });
+
+    it('should return uniq value of each item in array', () => {
+        expect(uniqArray(['hello', 'world', 'hello', 'world'])).toEqual([
+            'hello',
+            'world',
+        ]);
+    });
+
+    it('should return null if value is literal', () => {
```

Comment:
if value is NOT a literal, or better: if value is an object.

Reply:
I didn't change text from uppercase.spec.js, it's a copy/paste

Reply:
The text is right here too. And is `not an array` is even better